Framework/API/inc/MantidAPI/IMDHistoWorkspace.h

```diff
@@ -44,6 +44,10 @@ class DLLExport IMDHistoWorkspace : public IMDWorkspace,
   IMDHistoWorkspace_uptr clone() const {
     return IMDHistoWorkspace_uptr(doClone());
```

Comment:
Example:
If you have a `Workspace2D`, and you call `ws->clone()`, you want to get back a `Workspace2D`. Without covariance you would get a `Workspace`.